oml/utils/audios.py

```diff
@@ -0,0 +1,127 @@
+import base64
+from io import BytesIO
+from typing import Union
+
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+from PIL import Image
+from torch import FloatTensor
+
+from oml.const import BLACK, TColor
+
+
+def default_spec_repr_func(audio: FloatTensor) -> FloatTensor:
+    """
+    Generate a default spectral representation (log-scaled MelSpec) from an audio signal.
+    Used primarily for visualization.
+
+    Args:
+        audio: The input audio tensor.
+
+    Returns:
+        The spectral representation of the input audio tensor.
+    """
+    from torchaudio.transforms import MelSpectrogram
+
+    melspectrogram = MelSpectrogram(sample_rate=16000, n_fft=2048, hop_length=512, n_mels=128)
```

Comment:
Should this `sample_rate=16_000` somehow be synchronized with ours used for model training?

Reply:
at least we can use DEFAULT_SAMPLE_RATE
but ideally we should pass the actual sample rate
correct me if I'm wrong

Reply:
also, should we convert the hardcoded values to the function arguments with the default values?

Reply:
> Should this sample_rate=16_000 somehow be synchronized with ours used for model training?

This transform is solely used for visualisation, so it can't be used in training, because user would define own transform, which will be initialised only once somewhere near the dataset/model initialisation and applied during training loop.

i'll add DEFAULT_SAMPLE_RATE

Reply:
hm, okay, I'm not happy with that (I mean user may be confused by seeing a visualised spectogram with the SR different from the one used in his training), but let's leave this as it is in order to finish the PR faster :)